infra/src/main/java/com/redhat/service/bridge/infra/auth/AbstractOidcClient.java

```diff
@@ -0,0 +1,83 @@
+package com.redhat.service.bridge.infra.auth;
+
+import java.time.Duration;
+
+import javax.annotation.PostConstruct;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.redhat.service.bridge.infra.exceptions.definitions.platform.OidcTokensNotInitializedException;
+
+import io.quarkus.oidc.client.OidcClient;
+import io.quarkus.oidc.client.OidcClientConfig;
+import io.quarkus.oidc.client.OidcClientException;
+import io.quarkus.oidc.client.OidcClients;
+import io.quarkus.oidc.client.Tokens;
+
+public abstract class AbstractOidcClient {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AbstractOidcClient.class);
+
+    protected static final String SCHEDULER_TIME = "5s";
+    protected static final Duration SSO_CONNECTION_TIMEOUT = Duration.ofSeconds(30);
+    protected static final Duration REFRESH_TOKEN_TIME_SKEW = Duration.ofSeconds(30);
```

Comment:
Are them just overridable default values? If so it should be safe to make them `public`

Reply:
Only the subclasses that extend this class are going to eventually use those default values, so it should be fine to keep them protected